enterprise/admin/utils.py

```diff
@@ -112,3 +130,18 @@ def validate_email_to_link(email, raw_email=None, message_template=None, ignore_
             email=email, ec_name=existing_record.enterprise_customer.name
         ))
     return bool(existing_record)
+
+
+def get_course_runs_from_program(program):
+    """
+    Return course runs from program data.
+
+    Returns:
+        set: course runs in given program
+    """
+    return set(
```

Comment:
The logic here is a little hard to follow; I think it might be better to break this out into a set of nested clauses rather than a comprehension.